android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -364,6 +368,13 @@ public void run() {
     }
   }
 
+  private void fadeInListView() {
+    ObjectAnimator alphaAnimation = ObjectAnimator.ofFloat(getListView(), "alpha", 0, 1);
+    alphaAnimation.setDuration(400);
```

Comment:
Ah. This doesn't use an xml animation resource. It's purely programmatic. If that's what you're asking?